packages/web/src/scss/components/Button/README.md

```diff
@@ -27,7 +30,9 @@ Disabling a Button:
 Sizes:
 
 ```html
-<button type="button" class="Button Button--primary Button--medium">Medium Button</button>
+<button type="button" class="Button Button--primary Button--small">Small Button</button>
+<button type="button" class="Button Button--primary Button--small">Small Button</button>
```

Comment:
Not sure, but I think the pattern is "primary <SIZE>, inverted <SIZE>", so this line can be deleted.